extmod/mbedtls/mbedtls_config_common.h

```diff
@@ -79,6 +79,7 @@
 #define MBEDTLS_OID_C
 #define MBEDTLS_PKCS5_C
 #define MBEDTLS_PK_C
+#define MBEDTLS_PK_HAVE_ECC_KEYS
```

Comment:
Is this enabling a feature that wasn't enabled before?  Or is this needed to retain the same set of features with the new mbedTLS version?

Reply:
> needed to retain the same set of features with the new mbedTLS version

The latter.  Some code moved to a new (separate) file with previously available features now exposed by a compile-time macro, as I noted in my second comment above (https://github.com/micropython/micropython/pull/15547#issuecomment-2249692151)